src/candidates/profile/chat/ChatDialog.vue

```diff
@@ -124,6 +124,12 @@ export default {
   created () {
     if (!this.selectedContact) {
       const contact = _.find(this.contacts, (obj) => obj.id === this.$route.params.id);
+
+      if (!contact) {
+        this.$router.push({ name: 'ProfileChat' });
```

Comment:
Maybe we could additionally define constant for the router names, e.g. `PROFILE_CHAT_ROUTE`? That way if some route name needs to change, it won't have to be done manually across the project.

Reply:
Sounds good to me, but I would suggest to do it as a separate issue - we have few `this.$router.push()` scattered across the project and I think it would make sense to fix all of them.

Created #23 based on this.